misc/docerina/src/test/java/org/ballerinalang/docgen/tests/GenDocsForBalaTest.java

```diff
@@ -105,6 +105,20 @@ public void generatingDocsForBalaWithAnnotationTest() throws IOException {
         Assert.assertTrue(moduleApiDocsJsonAsString.contains("Task"), "Function annotation attachments missing");
     }
 
+    @Test
+    public void generatingDocsForBalaWithAnnotationTest2() throws IOException {
+        Path balaPath = this.resourceDir.resolve("balas").resolve("ballerina-http-java11-2.4.0.bala");
```

Comment:
Why did we add this test?

Reply:
This was added through different PR to master. Even my branch sync with latest master but compiletime-changes-for-function-defult-values didn't. So that changes were shown as well. We have sync with latest master. Now those commits are not shown